### PR TITLE
Fix vetting errors

### PR DESCRIPTION
**Pull request by fabxc**
@brian-brazil

### CODE REVIEW

File: notify/impl.go [hunk 1]
```diff
@@ -549,7 +549,7 @@ func (n *Hipchat) Notify(ctx context.Context, as ...*types.Alert) error {
 		data     = n.tmpl.Data(receiver(ctx), groupLabels(ctx), as...)
 		tmplText = tmplText(n.tmpl, data, &err)
 		tmplHTML = tmplHTML(n.tmpl, data, &err)
-		url      = fmt.Sprintf("%sv2/room/%s/notification?auth_token=%s", n.conf.APIURL, n.conf.RoomID, n.conf.AuthToken)
+		url      = fmt.Sprintf("%sv2/room/%d/notification?auth_token=%s", n.conf.APIURL, n.conf.RoomID, n.conf.AuthToken)
```
Comment (brian-brazil) [inline notify/impl.go]: RoomID is a string, %s sounds right to me

Comment (fabxc) [inline notify/impl.go]: Mh, that's odd. My editor show's it as type int in auto-completion and `go vet` did this:

```>> vetting code
notify/impl.go:552: arg n.conf.RoomID for printf verb %s of wrong type: int
```

Comment (brian-brazil) [inline notify/impl.go]: It was an int in the previous alertmanager, do you have a cache or the old version lying around or something?

Comment (fabxc) [inline notify/impl.go]: Nope, checked everything. Weird.
Changed it back.

**Pull request merged**